### DISCUSSION TITLE
Godot Remote Apk – Spanish Updated

### DISCUSSION THREAD

**Discussion by DualFidexGames** [godotengine/godot-proposals#9704] [Editor]
It would be very good if there was an apk called "Godot Remote" on mobile

Which allows you to control the project debugging screen. This would be good for developers who make mobile games. This way they don't have to export the project and transfer it to the mobile, to install it and then test it every time to see if it works or not...

Simply connect via WiFi (Local) or USB and you can share the debugging screen to your mobile and test it with your fingers on the mobile

**Comment by AThousandShips** [comment]
This is an English-speaking platform, please write in English (use machine translation if needed).

**Reply by DualFidexGames** [reply]
ok, tanks you

**Comment by Calinou** [comment]
It's likely technically feasible to deploy a PCK over the network and run it from a Godot Android app, but you'd need to restart the app every time the PCK is redeployed. We might be able to sidestep that if we integrate Godot as a library in the app (with the rest of the app being a native Java app, and therefore doesn't rely on Godot functionality to keep running).

This is pretty complex and hard to make robust in all situations though.
